index.css

```diff
@@ -0,0 +1,333 @@
+  border-radius: 8px;
+}
+
+.card-body.skills-body {
+  height: 158px;
+  background: none;
+}
+
+.list-view {
+  display: flex;
+  gap: 8px;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.list-item {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  padding: 8px 10px;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.list-item.skills-item {
+  color: #36b37e;
+  background: #fff;
+}
+
+h3.rcnt-h3 {
+  position: relative;
+  padding: 0 24px;
+  padding-bottom: 24px;
+  margin-bottom: 62px;
+}
+
+h2 > span {
+  color: rgb(37, 221, 114);
+}
+
+.rcnt-h3 > span {
+  position: absolute;
+  width: 48px;
+  height: 4px;
+  left: calc(50% - 48px / 2);
+  bottom: 0;
+  background: #36b37e;
+}
+
+.h2-logo {
+  font-size: 20px;
+  font-weight: 600;
+  color: #344563;
+}
+
+h5 {
+  line-height: 32px;
+  font-size: 24px;
+  font-weight: 600;
+  text-align: center;
+  color: #344563;
+  margin: 0;
+}
+
+h3 {
+  font-weight: 700;
+  font-size: 40px;
+  margin: 0;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+h2 {
+  font-weight: 700;
+  font-size: 40px;
+  list-style: 32px;
+  text-align: center;
+  color: #344563;
+  margin: 0;
+}
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+/* =================END===================================== */
+
+/* main layout */
+
+#main-layout {
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-l-item {
+  width: 100%;
+  max-width: 379px;
+}
+
+/* Header nav */
+
+.nav-bar {
+  display: flex;
+  padding: 10px 24px;
+  justify-content: space-between;
+}
+
+.nav-bar a {
+  font-weight: 600;
+  line-height: 28px;
+}
+
+/* ===== intro section ====== */
+
+.cont-preview {
+  padding-top: 64px;
+  padding-bottom: 89px;
+  position: relative;
+  background-image: url(./static/images/background.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.cont-desc {
+  margin: 58px 0;
+  padding: 0 20px;
+  text-align: center;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #172b4d;
+}
+
+/* social medias */
+
+.lets-connect {
+  padding: 0 20px;
+}
+
+.lets-connect ul {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  margin: 10px 0;
+}
+
+.socio-hub > img {
+  width: 18px;
+  height: 18px;
+}
+
+/* colapse button */
+
+.colapse-btn {
+  position: absolute;
+  width: 32px;
+  height: 32px;
+  left: calc(50% - 40px / 2 + 0.5px);
+  bottom: 4px;
+  border: 1px solid #dfe1e6;
+  border-radius: 32px;
+  background-image: url(./static/images/colpase-btn-icon.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* ====== My Recent Works =========== */
+
+.recent-works {
+  margin-top: 18px;
+}
+
+.works-container {
+  display: flex;
+  gap: 24px;
+  justify-content: center;
+  flex-wrap: wrap;
+}
```

Comment:
Please ensure that the main container of the projects has a `display:grid` according to the project requirement.

Reply:
Fixed